cloudinit/config/cc_set_passwords.py

```diff
@@ -238,4 +237,14 @@ def handle(_name, cfg, cloud, log, args):
 def rand_user_password(pwlen=9):
     return util.rand_str(pwlen, select_from=PW_SET)
 
+
+def chpasswd(distro, plist_in, hashed=False):
+    if util.is_FreeBSD():
+        for pentry in plist_in.splitlines():
+            u, p = pentry.split(":")
+            distro.set_passwd(u, p, hashed=hashed)
+    else:
+        cmd = ['chpasswd'] + ['-e'] if hashed else []
```

Comment:
This is bogus python of mine. Sorry. if hashed is False cmd becomes []. Silly order of precendence. this will fix your integration tests.
```suggestion
        cmd = ['chpasswd'] + (['-e'] if hashed else [])
```